docs/process/Estimation-Process.md

```diff
@@ -27,73 +27,130 @@ below the estimate. These assumptions translate to risks - because if
 
-Story point estimations are done by the development team together in
-the [[Meetings|sprint planning meeting]].
+This implies that:
 
-We use the [Pointing Poker](https://www.pointingpoker.com/) tool to
-facilitate estimation sessions.
+- The _Task_ ticket should have a complete set of acceptance criteria.
 
-The team should agree on a single number. If they don't then someone
-is missing some information about the task.
+- The _Task_ ticket should have a clear design that the pilot and co-pilot are
+  capable of estimating. Justification: there are many possible designs for a
+  given set of acceptance criteria. The time to deliver a task will depend very
+  strongly on the design that is chosen.
 
-This practice lets us share knowledge about tasks.
+- The _Task_ ticket should be linked to a _User Story_ ticket, **if** it
+  pertains to a user story. (It's important to note that not all tasks must
+  have a corresponding user story. In the case of technical debt tasks or pure
+  refactoring tasks, it might not be practical to even identify a user story.)
+
+- The _Task_ ticket is sufficiently small to estimate.
+
+- Task dependencies are defined with Jira issue links.
+
+## Process for updating estimates
+
+One of the main benefits of estimation is that both project management and other
+team members can gain visibility on how much time might be required to bring
+certain tasks to completion.
+
+Since we have one project planning meeting per week, it makes sense to update
+our estimates at around the same frequency.
+
+Before each project planning meeting (currently on Wednesday), each team member
+should:
+
+- Identify the tasks he or she is currently working on, in JIRA.
+
+- Make sure that each task has an updated pair of estimates that best reflect
+  their current understanding of how much remaining time would be required to
+  bring these tasks to completion.
+
+## Interpreting estimates
+
+Estimates do not represent commitments on behalf of those who create them.
+
+The actual time required to finish tasks, when compared to initial estimates,
+can vary wildly, with some tasks taking many times longer than initially
+predicted.
+
+This factor: the actual time divided by the estimated time, is often known as
+the "blowup factor".
+
+There is some research to indicate that the "blowup factor" (actual time
+divided by estimated time) can be reasonably modelled using a log-normal
+distribution. See https://erikbern.com/2019/04/15/why-software-projects-take-longer-than-you-think-a-statistical-model.html
+
+TODO:
+
+- [ ] When we have a task with a chain of dependent tasks, devise an automated
+  method of calculating a most-probable estimate and a most-pessimistic
+  estimate for the entire chain of tasks. This method might use the log-normal
+  distribution model suggested above in order to capture the possible blowup
+  factor for the entire chain.
+
+- [ ] When we have a release that contains large numbers of tasks with
```

Comment:
After reading the linked article, this sounds like it might prove useful for informing better commitment dates around intense bodies of work (like the PAB integration). I think we experienced (still are experiencing) pretty significant blowup factor for that work, which wasn't well represented by our initial estimates.